nncf/experimental/openvino_native/quantization/algorithms/fast_bias_correction/openvino_backend.py

```diff
@@ -104,6 +104,8 @@ def get_activation_port_ids_for_bias_node(node: NNCFNode) -> Tuple[int, int]:
 
     @staticmethod
     def is_quantized_weights(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        if node.layer_attributes is None:
```

Comment:
Not really, some nodes can have bias while don't have weights, e.g MatMul

Reply:
Could you please provide an example of MatMul with bias, but without weights? In my assumption, if the layer has weights, it may have bias, but not vice versa.

Reply:
![image](https://user-images.githubusercontent.com/32935044/225276860-2f10fd7c-1cec-4081-8a03-77812214ea23.png)

Reply:
This is not a bias. There is no constant for Add.

Reply:
You are right. But this line is needed as we don't have a check whether the node has a weight before